src/diabetes_predictor/data_processor.py

```
@@ -0,0 +1,18 @@
+        self.config = config
+        self.X = None
+        self.y = None
+        self.preprocessor = None
+
+    def load_data(self, filepath):
+        return pd.read_csv(filepath)
+    
+    def preprocess_data(self):
+        # Remove rows with missing target
+        target = self.config['target']
+        self.df = self.df.dropna(subset=[target])
```

Ruff

No newline at end of file

Add trailing newline

(W292)